tools/automate_gcp_os_version.py

```diff
@@ -0,0 +1,118 @@
+"""
+checks for the latest os and updates the os versions in vars.tf file in dc-only, 
+multi-region and single connector deployments.
+Use this command to run this script in cloud shell: python3 tools/update_gcp_os.py
+"""
+import argparse
+import datetime
+import os
+import re
+import subprocess
+from subprocess import CalledProcessError
```

Comment:
just a personal pref: there is only 1 use and you use import subprocess already, could use subprocess.CalledProcessError where it is used.